plugins/pencilblue/controllers/api/admin/site_settings/email/send_test.js

```diff
@@ -16,37 +16,44 @@
 */
 
 module.exports = function(pb) {
-    
+
     //pb dependencies
     var util = pb.util;
-    
+
     /**
      * Resends an account verification email
      */
     function SendTestEmail(){}
-    util.inherits(SendTestEmail, pb.FormController);
+    util.inherits(SendTestEmail, pb.BaseController);
 
-    SendTestEmail.prototype.onPostParamsRetrieved = function(post, cb) {
+    SendTestEmail.prototype.render = function(cb) {
         var self = this;
 
-        var message = this.hasRequiredParams(post, this.getRequiredFields());
-        if(message) {
-            cb({content: pb.BaseController.apiResponse(pb.BaseController.API_FAILURE, message)});
-            return;
-        }
-
-        var options = {
-            to: post.email,
-            subject: 'Test email from PencilBlue',
-            layout: 'This is a successful test email from the PencilBlue system.',
-        };
-        var emailService = new pb.EmailService();
-        emailService.sendFromLayout(options, function(err, response) {
-            if(err) {
-                cb({content: pb.BaseController.apiResponse(pb.BaseController.API_FAILURE, JSON.stringify(err))});
+        this.getJSONPostParams(function(err, post) {
```

Comment:
Don't forget that you can declare the incoming content type in the route definition and just call ```this.body```.  It will save you a lot of space and reduce complexity in the controllers